src/Turtle/Format.hs

```diff
@@ -107,21 +108,31 @@ instance (a ~ b) => IsString (Format a b) where
 format :: Format Text r -> r
 format fmt = fmt >>- id
 
+{-| Create your own formatted printer function
+
+Can be useful for making custom "printf" and/or "eprintf" functions that would
```

Comment:
This will ensure that the haddocks link to their definitions:

```suggestion
Can be useful for making custom 'printf' and/or 'eprintf' functions that would
```

Reply:
@Gabriel439 Referring to the Haddock documentation:

https://haskell-haddock.readthedocs.io/en/latest/markup.html#linking-to-modules

> ### Linking to Modules
>  
> Linking to a module is done by surrounding the module name with double quotes:
>  
> `-- | This is a reference to the "Foo" module.`

Reply:
@Gabriel439 ah, sorry, it’s linking to *modules*, not function names.